### PR TITLE
GH-2851 Docs for PayingForTx

### PR DESCRIPTION
**Pull request by hanssv**
Associated to https://github.com/aeternity/aeternity/pull/2999

Also fixes some obviously incorrect values.

### CODE REVIEW

File: consensus/consensus.md [hunk 1]
```diff
@@ -274,14 +274,31 @@ both the `Fee` and the `GasPrice`.
  -------------- -----
 | recipient    | 32  |
  -------------- -----
-| amount       | 8   |
+| amount       | var |
```
Comment (tolbrino) [inline consensus/consensus.md]: `var` should be defined somewhere.

File: consensus/consensus.md [hunk 2]
```diff
@@ -339,7 +356,9 @@ The gas of a transaction is the sum of:
...
 | Contract call (AEVM)   | `30 * BaseGas` | Proportional to the byte size of the transaction, specifically: `byte_size(ContractCallTx) * GasPerByte`. It also includes gas for contract execution. |
 | GA Attach              | `5 * BaseGas`  | Proportional to the byte size of the transaction, specifically: `byte_size(GAAttachTx) * GasPerByte`. It also includes gas for execution of the init function. |
-| GA Meta                | `5 * BaseGas`  | Proportional to the byte size of the transaction, specifically: `byte_size(GAMetaTx) * GasPerByte`. It also includes gas for execution of the authentication function + recursively gas corresponding to the wrapped transaction(s) (excluding the byte size portion - in order not to account for the size of wrapped transactions multiple times). |
+| GA Meta                | `5 * BaseGas`  | Fortuna and Lima : Proportional to the byte size of the transaction, specifically: `byte_size(GAMetaTx) * GasPerByte`. It also includes gas for execution of the authentication function + recursively gas corresponding to the wrapped transaction(s) (excluding the byte size portion - in order not to account for the size of wrapped transactions multiple times). |
```
Comment (tolbrino) [inline consensus/consensus.md]: Reads like this is not available post-Lima.

File: serializations.md [hunk 3]
```diff
@@ -995,6 +996,20 @@ authorization data.
 ]
 ```
 
+### PayingFor
+The `PayingFor` transaction is available from version 5, Iris release. By using
+it an account `P` can pay for the transaction (transaction fee + gas) of another
```
Comment (ThomasArts) [inline serializations.md]:
```suggestion
it,  an account `P` can pay for the transaction (transaction fee + gas) of another
```

File: consensus/consensus.md (hunk 1 above)
Comment (hanssv) [reply consensus/consensus.md]: Hmm, I just saw it being used further up in the same document so I assumed it to be defined...

File: consensus/consensus.md (hunk 2 above)
Comment (hanssv) [reply consensus/consensus.md]: So I guess I failed then _"I tried to make it obvious this isn't around until IRIS"_

But in a sense it isn't available in Iris, the next definition applies then...

File: consensus/consensus.md (hunk 1 above)
Comment (tolbrino) [reply consensus/consensus.md]: That is why I approved nonetheless.

Comment (hanssv) [reply consensus/consensus.md]: 😅

**Pull request merged**